### DISCUSSION TITLE
Rimozione Entità (binary.sensor) impossibile!  @@ RISOLTO @@

### DISCUSSION THREAD

**Discussion by Jones1982** [indomus/forum#21] [HUB: Home Assistant]
Buongiorno,
non riesco a cancellare un entità "Sirena Numero Melodia" (vedi screenshot) fatta tramite Local Tuya da Integrazioni!.

Ho provato anche a ricrearla con lo stesso nome, ma crea un id entità con numerazione progressiva.

![b1525083-09b5-43a6-816a-9fd77af4ee6a-sirena](https://user-images.githubusercontent.com/57654443/112465978-9a9b1480-8d65-11eb-9332-ee9447ae9115.PNG)

Come posso rimuoverlo? grazie a tutti!

P.s: nonostante ho rimosso il dispositivo dall'app Smart Life (tuya), continua a visualizzare questa entità, e non c'è una voce per rimuoverla

**Comment by cobradodo** [comment]
Ciao, ora non ho sottomano il mio HA, però mi era capitato... per risolvere avevo guardato in alcuni file contenuti dentro la cartella .storage
Forse core.restore_state...
facendo occhio ad eliminare il blocco da { a } e se è l'ultimo eliminando la virgola...

Ciao

**Reply by Jones1982** [reply]
Ciao, grazie per la risposta!
Ho aperto il file che hai detto tu, e ho cercato l'entità che voglio rimuovere.

Questo è il rigo che c'è nel file, è contenuto in 4 parentesi graffe, quindi desumo devo rimuovere tutto, giusto?
Ci sono anche spazi all'inizio e alla fine dopo la virgola, quelli devo lasciarli come se non fosse mai esistito il rigo!

Confermi? grazie

Ovviamente ho salvato il file intatto sul pc, prima di applicare la modifica.

```
        {
            "state": {
                "entity_id": "binary_sensor.sirena_numero_melodia",
                "state": "unavailable",
                "attributes": {
                    "friendly_name": "Sirena Numero Melodia"
                },
                "last_changed": "2021-03-28T20:26:03.604721+00:00",
                "last_updated": "2021-03-28T20:26:03.604721+00:00",
                "context": {
                    "id": "ac6d9e535ed15a5a73592bdfc70d4690",
                    "parent_id": null,
                    "user_id": null
                }
            },
            "last_seen": "2021-03-29T08:11:12.444401+00:00"
        },
```

Aggiornamento: non è cambiato nulla, e in questo file è ricomparsa l'entità.
Credo che il file corretto in questione sia core.entity_registry   !?

Aggiornamento 2:  era il file core.config_entries
Gli altri tengo solo traccia dello stato ecc..
**TUTTO OK**

**### _GRAZIE!!!_**

**Reply by cobradodo** [reply]
perfetto :-D si andavo a memoria... però il concetto era di cercare in quella cartella
